plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -387,7 +387,6 @@ func processOsSignals(cancelFunc context.CancelFunc) {
 func (rsl *RiemannSocketListener) Stop() {
 	rsl.wg.Done()
 	rsl.wg.Wait()
-	os.Exit(0)
```

Comment:
not sure why this is here, though it looks like a good change.

Reply:
It was causing the build to fail, I can move it to a separate pr though.

Reply:
This `os.Exit()` should definitively be killed. If we see problems I think something else is fundamentally wrong, as, if I understand this correctly, telegraf will just die once it stops the first instance of `RiemanSocketListener`, taking away the chance for all other (output) plugins to close their connection to their sink.